local-cli/eject/eject.js

```diff
@@ -84,6 +126,54 @@ function eject() {
       appName,
       templateOptions
     );
+
+    console.log('Setting up App Icons');
```

Comment:
can the log mention the platform of the icons being set up? Same with the iOS log